src/main/webapp/scripts/comments.js

```diff
@@ -0,0 +1,25 @@
+export { makeApiCall, buildRequestBody };
+
```

Comment:
Should the file name be post-comment? Comparing to test file.

Reply:
Done.